webapp/app/plugins/controller.js

```diff
@@ -138,7 +138,8 @@ treeherder.controller('PluginCtrl', [
                         }
                     });
                     // retry to fetch the job info if a log hasn't been parsed yet
-                    if(logsNotParsed.length > 0){
+                    // but only if a job is still selected
+                    if(logsNotParsed.length > 0 && $scope.selectedJob){
```

Comment:
I think the potential for a race (ie someone deselecting the job, so no job is selected) is more likely after this point, not before? ie: should the check for a selected job be wrapping selectJobAndRender() below - since that gets called after the 5 seconds? Or even better, could we cancel the setTimeout once the panel becomes deactivated?